.github/CODEOWNERS

```diff
@@ -543,10 +543,12 @@
 /.lighthouserc.json @jimbyo @jameesjohn @vojtechjelinek
 /puppeteer-login-script.js @jimbyo
 /scripts/run_lighthouse_tests.py @jameesjohn @vojtechjelinek
-/webpack.common.config.ts @jameesjohn @vojtechjelinek
+/webpack.common.config.ts @nishantwrp @jameesjohn @vojtechjelinek
 /webpack.common.macros.ts @jameesjohn @vojtechjelinek
-/webpack.dev.config.ts @jameesjohn @vojtechjelinek
-/webpack.prod.config.ts @jameesjohn @vojtechjelinek
+/webpack.dev.config.ts @nishantwrp @jameesjohn @vojtechjelinek
+/webpack.dev.sourcemap.config.ts @nishantwrp @vojtechjelinek
+/webpack.prod.config.ts @nishantwrp @jameesjohn @vojtechjelinek
+/webpack.prod.sourcemap.config.ts @nishantwrp @vojtechjelinek
 /webpack.terser.config.ts @nishantwrp @vojtechjelinek
```

Comment:
Let's do `/webpack.*.ts` here with me and you as code owners.

Reply:
done